To: vim_dev@googlegroups.com
Subject: Patch 7.3.130
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.130
Problem:    Variable misplaced in #ifdef.
Solution:   Move clipboard_event_time outside of #ifdef.
Files:	    src/gui_gtk_x11.c


*** ../vim-7.3.129/src/gui_gtk_x11.c	2010-08-15 21:57:31.000000000 +0200
--- src/gui_gtk_x11.c	2010-12-04 15:26:41.000000000 +0100
***************
*** 86,94 ****
  
  #ifdef HAVE_X11_SUNKEYSYM_H
  # include <X11/Sunkeysym.h>
- static guint32 clipboard_event_time = CurrentTime;
  #endif
  
  /*
   * Easy-to-use macro for multihead support.
   */
--- 86,95 ----
  
  #ifdef HAVE_X11_SUNKEYSYM_H
  # include <X11/Sunkeysym.h>
  #endif
  
+ static guint32 clipboard_event_time = CurrentTime;
+ 
  /*
   * Easy-to-use macro for multihead support.
   */
***************
*** 5419,5425 ****
  	}
  
  #if defined(FEAT_NETBEANS_INTG)
! 	/* Process the queued netbeans messages. */
  	netbeans_parse_messages();
  #endif
  
--- 5420,5426 ----
  	}
  
  #if defined(FEAT_NETBEANS_INTG)
! 	/* Process any queued netbeans messages. */
  	netbeans_parse_messages();
  #endif
  
*** ../vim-7.3.129/src/version.c	2011-02-25 15:13:43.000000000 +0100
--- src/version.c	2011-02-25 15:15:02.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     130,
  /**/

-- 
A vacation is a period of travel during which you find that you
took twice as many clothes and half as much money as you needed.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
